src/dynamic-pages/analyze/sections/5-Contributors.tsx

```diff
@@ -252,15 +252,19 @@ export const Contributors = forwardRef(function ({}, ref: ForwardedRef<HTMLEleme
       exclusive
       onChange={handleChangeType}
     >
-      <ToggleButton value='count'>Count</ToggleButton>
-      <ToggleButton value='percentage'>Percentage</ToggleButton>
+      <ToggleButton value='count'># Total Events</ToggleButton>
+      <ToggleButton value='percentage'>% Percentage</ToggleButton>
     </ToggleButtonGroup>
   )
 
   return (
     <Section id="contributors" ref={ref}>
-      <H2>Contributors</H2>
-      <P2>Contribution status in multiple dimensions LAST MONTH</P2>
+      <H2>Contributor Rankings - 2022/05</H2>
```

Comment:
Need to be replaced with dynamic data @634750802

Reply:
+1